service/build.gradle

```diff
@@ -129,7 +129,7 @@ dependencies {
     implementation "org.glassfish.jersey.media:jersey-media-multipart:${gradle.vJersey2}"
     implementation "org.glassfish.jersey.media:jersey-media-json-jackson:${gradle.vJersey2}"
 
-    implementation 'org.vumc.vda.tanagra:admin-client:0.0.85-SNAPSHOT'
+    implementation 'org.vumc.vda.tanagra:admin-client:0.0.98-SNAPSHOT'
```

Comment:
re: https://github.com/DataBiosphere/tanagra/blob/main/service/build.gradle#L125
Jersey libraries were kept at v2 for java 11 client. These can be updated to v3. Common versions set and defined here - https://github.com/DataBiosphere/tanagra/blob/main/build.gradle

Reply:
I tried change to v3- but other dependencies broke for tanagara-admin hence will do it later.